src/websocket_tracker_connection.cpp

```diff
@@ -174,13 +174,13 @@ void websocket_tracker_connection::send_pending()
 	auto [msg, callback] = std::move(m_pending.front());
 	m_pending.pop();
 
-	std::visit([&](auto const& m)
+	std::visit([callback = std::move(callback)](auto const& m)
```

Comment:
this change is causing a compilation error for me (which you may have noticed by now). you can't assign `m_requester` without capturing `this`. But I think the previous code was find, capturing everything makes sense in a `visit()` call I would say

Reply:
Yeah I misunderstood the warning `error: reference to local binding 'callback' declared in enclosing function`. The problem is that strictly-speaking bindings are not default-capturable because they are not variables.